Mollie/API/Client.py

```diff
@@ -69,9 +74,10 @@ def performHttpCall(self, http_method, path, data=None, params=None):
         url = '%s/%s/%s' % (self.api_endpoint, self.api_version, path)
         data = '{}' if data is None else data
 
-        query_string = generate_query_string(params)
-        url += '?' + query_string
-        params = None
+        query_string = generate_querystring(params)
+        if query_string:
```

Comment:
dit wil je niet in de generate_query_string stppen?

Reply:
nee, want dan moet ik de url en de params ook allemaal meegeven. Dat maakt het allemaal weer nodeloos gecompliceerd, en lastiger te testen, IMHO

Reply:
ok